data/competencies.yml

```diff
@@ -925,3 +925,15 @@
   level: senior1-to-senior2
   area: leadership
   domain: null
+
+
+- id: f173b7e9-1c46-4afd-a659-d264aa173759
+  summary: Contributes to inclusive organisational culture
+  examples:
+    - Membership of internal groups that promote diversity and inclusion (such as FT Embrace, FT Proud, Mental health first aiders)
```

Comment:
Could consider adding 'FT Access' to this list 😇

Reply:
Agreed. I've done this as part of a suggestion above